tests/reporters/test_python_native_reporter.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from approvaltests import verify
+
+from approvaltests.reporters.python_native_reporter import *
+
+
+def test_files_identical(tmpdir):
```

Comment:
I don't think that these tests will be run in CI. The tests are run using Python's `unittest` discovery, `python -m unittest discover -v -s . -p "*.py"`, see `tox.ini`.
I've just added the verbose flag, `-v`, so that we'll be able to see the tests that are run in the stdout on travis-ci. Rebasing from master should pull this in allow you to see the outcome here: https://travis-ci.org/approvals/ApprovalTests.Python/pull_requests